catkin_ws/src/state_estimation/src/state_aggregator.py

```diff
@@ -7,65 +7,72 @@
 from tf2_ros import TransformBroadcaster
 
 from geometry_msgs.msg import Pose, Quaternion, Vector3, TransformStamped, Point
-from sensors import DepthSensor, IMU, DVL
-from std_msgs.msg import Float64, Bool
+from sensors import DepthSensor, IMU, IMUFrontCamera, DVL
+from std_msgs.msg import Float64, Int32
 from rosgraph_msgs.msg import Clock
 
 DEG_PER_RAD = 180 / np.pi
 
+def swap_main_sensor_message(axis_index, sensor_name):
```

Comment:
the function name had me hella confused, could we just call it "print_sensor_source" or something? Also im not sure if its best to have it check for 5 seconds inside the function but check for axis !=0 outside, might as well have all logic related to logging inside (function calls take like 0.0000001 seconds)